### ISSUE TITLE
Imperfectly registered special fonts

### ISSUE THREAD

**Issue by Kwakoxium** [Tyrrrz/DiscordChatExporter#762]
### Version

2.31.1

### Flavor

GUI (Graphical User Interface)

### Export format

HTML

### Details

Hi!

I noticed a little oddity. Seeing people playing with various special fonts, I wanted to do a little test by recording and I could see some anomalies. The characters on the same line are at two different heights. Characters on the same line are on two different heights. Accented characters have their accents going in all directions.

**Here's what appears on Discord:**
![Special fonts  1](https://user-images.githubusercontent.com/87956094/147892867-8b41f914-20ff-4d59-a35e-79e0d27b10c6.png)

**Here is what it looks like when saved in HTML:**
![Special fonts  2](https://user-images.githubusercontent.com/87956094/147892874-59992887-612a-468d-b524-37820b8d6fa3.png)

Thank you for your help!

### Steps to reproduce

Put simply, people were using this site to convert their normal texts to texts with a special font: https://www.messletters.com/en/. There is also a significant amount if that helps.

**Comment by Tyrrrz** [comment]
Can you please copy-paste the exact message here so I can debug it?

Use fenced blocks to avoid formatting.

**Comment by CanePlayz** [comment]
For the sake of completeness and archival purposes: I couldn't reproduce this.
